docs/source/design_notation.md

```diff
@@ -0,0 +1,90 @@
+
+## Pretest-posttest designs
+
+One of the simplest designs is the pretest-posttest design. Here we have one row, denoting a single group of units. There is an `X` which means all are treated. The pretest is denoted by $O_1$ and the posttest by $O_2$. See p99 of Reichardt (2019).
+
+|    |   |    |
+|----|---|----|
+$O_1$ | X | $O_2$ |
+
+Informally, if we think about drawing conclusions about the causal impact of the treatment based on the change from $O_1$ to $O_2$, we might say that the treatment caused the change. However, this is a tenuous conclusion because we have no way of knowing what would have happened in the absence of the treatment.
+
+A variation of this design which may (slightly) improve this situation from the perspective of making causal claims, would be to take multiple pretest measures. This is shown below, see p107 of Reichardt (2019).
+
+|    |  |   |    |
+|----|--|---|----|
+$O_1$ | $O_2$ | X | $O_3$ |
+
+This would allow us to estimate how the group was changing over time before the treatment was introduced. This could be used to make a stronger causal claim about the impact of the treatment.
+
+## Nonequivalent group designs
+
+In randomized experiments, with large enough groups, the randomization process should ensure that the treatment and control groups are equivalent. However, in quasi-experimental designs, with non-random (`NR`) allocation, we could expect there to be differences between the treatment and control groups. This poses some challenges in making strong causal claims about the impact of the treatment.
+
+For example, in the simplest nonequivalent group design, we have two groups, one treated and one not treated, and just one posttest. See p114 of Reichardt (2019).
+
+|     |   |    |
+|-----|---|----|
+| NR: | X | $O_1$ |
+| NR: |   | $O_1$ |
+
+The above design would be considered weak - the lack of a pre-test measure makes it hard to know whether differences between the groups at $O_1$ are due to the treatment or to pre-existing differences between the groups.
+
+This limitation can be addressed by adding a pretest measure. See p115 of Reichardt (2019).
+
+|     |    |   |    |
+|-----|----|---|----|
+| NR: | $O_1$ | X | $O_2$ |
+| NR: | $O_1$ |   | $O_2$ |
+
+Non-equivalent group designs like this, with a pretest and a posttest measure could be analysed in a number of ways:
+1. **ANCOVA:** Here, the group would be a categorical predictor, the pretest measure would be a covariate, and the posttest measure would be the outcome.
```

Comment:
I think is important to say is not *any* covariate, but actually the needed onws to be able to have all backdor paths closes (or maybe even stick with the potential outcomes notation)

Reply:
So I'm leaving the potential outcome and DAG approaches for other docs pages. So what I've done for the moment is edit the text to say:

> Here, the group would be a categorical predictor (e.g. treated/untreated), the pretest measure would be a covariate (though there could be more than one), and the posttest measure would be the outcome.

Let me know what you think.

Reply:
given the scope of this doc (no potential outcomes and no DAGs), then is look ok 👍